bot/cogs/snakes.py

```diff
@@ -40,6 +69,28 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        embed = discord.Embed(color=0x3E885B)
+        if name == "python":
+            # handle Python special case
+            embed.add_field(
+                name="Python (programming language)",
+                value="*Guido van Rossum*\n\n"
```

Comment:
multiline implicit string joins should be wrapped inside paranthesis, so that it's clear that they're one thing. This is a bit too implicit.

Reply:
Can you explain this? I'm not super clear as to what you mean...

Reply:
```py
value=(
    "*Guido van Rossum*\n\n"
    "This language is neither dangerous nor venomous and be found in software globally"
),
```
this is one way.

or if you prefer it, this is acceptable too
```py
value=("*Guido van Rossum*\n\n"
       "This language is neither dangerous nor venomous and be found in software globally"),
```